### PR TITLE
readme-updates

### PR DESCRIPTION
**Pull request by matschaffer**
Finally got this working so noted down a few more bits to hopefully make it smoother for others in the future.

### CODE REVIEW

**Review by ruflin — APPROVED** [review]
LGTM. It has been a while since I set it up so didn't go through the flow but what you put down here makes a lot of sense.

**Comment by matschaffer** [comment]
Thanks! Looks like I lack a merge button here so will leave it to you or @graphaelli

**Comment by ruflin** [comment]
No merge button for me either 😢

File: README.md
Comment (graphaelli) [inline README.md]:
```suggestion
```

**Review by graphaelli — CHANGES_REQUESTED** [review]

**Review by graphaelli — APPROVED** [review]
Thanks!

**Pull request merged**
